packages/venia-concept/package.json

```diff
@@ -24,14 +24,16 @@
     "start:debug": "node --inspect-brk ./node_modules/.bin/webpack-dev-server --progress --color --env.mode development",
     "test": "jest src",
     "validate-queries": "node ./validate-queries.js",
-    "watch": "webpack-dev-server --progress --color --env.mode development"
+    "watch": "webpack-dev-server --progress --color --env.mode development",
+    "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook:build": "build-storybook -c .storybook -o storybook-dist"
   },
   "dependencies": {
     "@magento/eslint-config": "~1.2.3",
     "@magento/peregrine": "^2.0.0-rc.15",
     "@magento/pwa-buildpack": "^2.0.0-rc.15",
     "@magento/upward-js": "^2.0.0-rc.15",
-    "@storybook/react": "~3.4.2",
+    "@storybook/react": "~4.0.11",
```

Comment:
Last time we attempted this, we ran into `now.sh` issues with Babel and [had to rip it out](https://github.com/magento-research/pwa-studio/pull/540).

@zetlen does the fact that the `now` check completed successfully mean that this PR does not have the same issue?

For reference, we do have [a PR for upgrading to Babel 7](https://github.com/magento-research/pwa-studio/pull/490).

Reply:
@supernova-at I do believe that's what it means!